nautobot/extras/tests/test_api.py

```diff
@@ -3242,7 +3242,7 @@ def test_all_relevant_content_types_assigned_to_tags_with_empty_content_types(se
         tag = Tag.objects.get(slug=self.create_data[0]["slug"])
         self.assertEqual(
             tag.content_types.count(),
-            TaggableClassesQuery().as_queryset.count(),
+            TaggableClassesQuery().as_queryset().count(),
```

Comment:
Looks like there are a couple of additional cases of `as_queryset` at lines 3268 and 3271 that need to be updated.